Makefile

```
@@ -159,7 +158,7 @@ test-ext-dev-e2e: $(OPERATOR_SDK) $(KUSTOMIZE) $(KIND) #HELP Run extension creat
 	go test -count=1 -v ./test/extension-developer-e2e/...
 
 ENVTEST_VERSION := $(shell go list -m k8s.io/client-go | cut -d" " -f2 | sed 's/^v0\.\([[:digit:]]\{1,\}\)\.[[:digit:]]\{1,\}$$/1.\1.x/')
-UNIT_TEST_DIRS := $(shell go list ./... | grep -v /test/)
+UNIT_TEST_DIRS := $(shell go list ./... | grep -v /test/ | grep -v catalogd/test/e2e | grep -v catalogd/test/upgrade)
```

Can this be simplified?

```suggestion
UNIT_TEST_DIRS := $(shell go list ./... | grep -v /test/ | grep -v catalogd/test)
```

---

fixed